samples/modules/lvgl/demos/boards/da1469x_dk_pro.conf

```diff
@@ -0,0 +1,16 @@
+#
+# Copyright (c) 2023 Renesas Electronics Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+#
+CONFIG_HEAP_MEM_POOL_SIZE=4096
+CONFIG_LOG_MODE_MINIMAL=y
+CONFIG_MIPI_DBI_LOG_LEVEL_DBG=y
+CONFIG_DISPLAY_LOG_LEVEL_DBG=n
+CONFIG_INPUT_FT5336_INTERRUPT=y
+CONFIG_LV_Z_POINTER_INPUT_MSGQ_COUNT=70
+# Frame buffer must always be 4-byte aligned. This is imposed by LCDC.
+CONFIG_LV_Z_VDB_ALIGN=4
```

Comment:
You might consider setting these values in the board's Kconfig.defconfig, since they will be needed by customers using LVGL as well. https://github.com/zephyrproject-rtos/zephyr/blob/7f5b332b58a9bfbbc6ed7b91e1bfc03f5f0ca7f7/boards/arm/mimxrt1060_evk/Kconfig.defconfig#L61